modules/ROOT/pages/directives/custom-logic.adoc

```diff
@@ -334,6 +334,11 @@ type Movie @limit(max: 100, default: 10) @node {
 
 == `@customResolver`
 
+[NOTE]
```

Comment:
👏 

It's worth noting that there's a few other directives that are not supported currently.

The list is `@populatedBy`, `@customResolvers`, @subscriptions`, `@subscriptionsAuthorization` & `@vector`

Ideally we should add support for the last 3 soon though!